src/openloco/windows/industry_list.cpp

```diff
@@ -1,16 +1,1272 @@
+                if (yPos + rowHeight < dpi->y || yPos >= yPos + rowHeight + dpi->height || industryId == (uint16_t)-1)
+                {
+                    yPos += rowHeight;
+                    continue;
+                }
+
+                string_id text_colour_id = string_ids::white_stringid2;
+
+                // Highlight selection.
+                if (industryId == self->row_hover)
+                {
+                    gfx::draw_rect(dpi, 0, yPos, self->width, rowHeight, 0x2000030);
+                    text_colour_id = string_ids::wcolour2_stringid2;
+                }
+
+                if (industryId == 0xFFFF)
+                    continue;
+                auto industry = industrymgr::get(industryId);
+
+                // Industry Name
+                {
+                    auto args = FormatArguments();
+                    args.push(industry->name);
+                    args.push(industry->town);
+
+                    gfx::draw_string_494BBF(*dpi, 0, yPos, 198, colour::black, text_colour_id, &args);
+                }
+                // Industry Status
+                {
+                    const char* buffer = stringmgr::get_string(string_ids::buffer_1250);
+                    industry->getStatusString((char*)buffer);
+
+                    auto args = FormatArguments();
+                    args.push(string_ids::buffer_1250);
+
+                    gfx::draw_string_494BBF(*dpi, 200, yPos, 238, colour::black, text_colour_id, &args);
+                }
+                // Industry Production Delivered
+                {
+                    if (!industry->canProduceCargo())
+                    {
+                        yPos += rowHeight;
+                        continue;
+                    }
+
+                    auto productionTransported = getAverageTransportedCargo(*industry);
+
+                    auto args = FormatArguments();
+                    args.push<uint16_t>(productionTransported);
+
+                    gfx::draw_string_494BBF(*dpi, 440, yPos, 138, colour::black, string_ids::production_transported_percent, &args);
+                }
+                yPos += rowHeight;
+            }
+        }
+
+        // 0x00458113
+        static ui::cursor_id cursor(window* self, int16_t widgetIdx, int16_t xPos, int16_t yPos, ui::cursor_id fallback)
+        {
+            if (widgetIdx != widx::scrollview)
+                return fallback;
+
+            uint16_t currentIndex = yPos / rowHeight;
+            if (currentIndex < self->var_83C && self->row_info[currentIndex] != -1)
+                return cursor_id::hand_pointer;
+
+            return fallback;
+        }
+
+        // 0x004580DE
+        static void event_08(window* self)
+        {
+            self->flags |= window_flags::flag_14;
+        }
+
+        // 0x004580E6
+        static void event_09(window* self)
+        {
+            if ((self->flags & window_flags::flag_14) == 0)
```

Comment:
This flag is set only when not on a scroll view. I think we should be able to come up with a name for it.